packages/frontend2/src/icons/pages/Finality.tsx

```diff
@@ -0,0 +1,67 @@
+import { cn } from '~/utils/cn'
+
```

Comment:
file name should be FinalityIcon

Reply:
btw. why not svg?

Reply:
I named it `Finality.tsx` because it's an icon and has an `export default`, but it's tsx because of some logic that exists outside - on darkmode the icon is different.